docs/source/examples/links.rst

```diff
@@ -29,6 +29,13 @@ There are a few links you can use with different purposes.
           .. _here:
      - .. _here:
      - This is an internal cross reference bookmark. It requires an internal cross-reference anchor (above). It does not render, but serves as a point to link to.
+   * - Internal Cross-reference to Parameters
+     - .. code-block:: rst
+
+          :ref:`Link to a config param <confprop_parametr_name>`
+     - Renders the parameter name as a hyperlink.
+     - This is an internal cross-reference to a parameter on the `Configuration Parameters <https://opensource.docs.scylladb.com/stable/reference/configuration-parameters.html>`_ page.
+       It does NOT require a bookmark, as parameter names on that are page implicit bookmarks. Content opens in the same tab.  
    * - Internal Doc Reference
```

Comment:
We could document how to link to a group of parameters too. From https://github.com/scylladb/scylladb/pull/17753:

> Additionally, it also adds support for linking groups. For example, ":ref:Ungrouped properties <confgroup_ungrouped_properties>".